i18n/translations/pt-br.json

```diff
@@ -1,90 +1,219 @@
 {
   "@metadata": {
-    "authors": [
-      "Eduardo Addad de Oliveira"
-    ]
+      "authors": [
+          "Eduardo Addad de Oliveira",
+          "luandro"
+      ]
   },
+  "a_new_firmware_version_has_been_released_17f89266": "Uma nova versão do firmware foi lançada",
+  "admin_4d3ba42f": "Administrador",
+  "admin_password_6b9d0812": "Senha",
   "align_11050992": "Alinhar",
+  "an_error_occurred_a4e1cda4": "Ocorreu um erro",
+  "applying_changes_23ae34f2": "Aplicando mudanças",
+  "applying_changes_the_system_takes_a_minute_to_get__189a700d": "Aplicando alterações, o sistema leva um minuto para voltar.",
+  "ask_for_remote_support_7e7eaab0": "Pedir suporte remoto",
+  "at_least_one_alphanumeric_character_357817ee": "Ao menos um caractere alfanumérico",
+  "at_least_one_number_bf8434bb": "Ao menos um número",
   "back_to_base_443797cb": "Voltar à base",
   "base_host_a17d45a4": "Servidor de base",
+  "best_signal_913e3460": "Melhor sinal",
+  "cancel_caeb1e68": "Cancelar",
+  "cancel_cd471b5e": "Cancelar",
+  "cannot_connect_to_the_remote_support_server_6a95528b": "Não foi possível conectar ao servidor de suporte remoto",
+  "cannot_load_map_check_your_internet_connection_d24f5daf": "Não foi possível carregar o mapa, verifique sua conexão",
   "change_dcaa253a": "Mudar",
-  "connected_host_91e11459": "Host conectado",
+  "change_shared_password_58dc580c": "Mudar senha compartilhada",
+  "checking_connection_863b319e": "Verificando conexão",
+  "choose_a_name_for_this_node_f491aa8d": "Escolha um nome para esse nó",
+  "choose_a_name_for_your_network_69df41e3": "Escolha um nome para a sua rede",
+  "choose_a_shared_password_for_network_administration_f182ed3b": "Escolha uma senha compartilhada para administração da rede",
+  "click_at_close_session_to_end_the_remote_support_s_1af24bb9": "Clique em Encerrar Sessão para finalizar a sessão de suporte remoto. Ninguém vai mais poder acessar seu nó usando essa chave novamente.",
+  "click_at_show_console_to_follow_the_remote_support_8b39ccbc": "Clique em Mostrar Console para acompanhar a sessão de suporte remoto",
+  "close_session_89aadfa2": "Encerrar Sessão",
+  "cofirm_upgrade_before_seconds_seconds_or_it_will_b_48b600b6": "Confirme a atualização antes de %{seconds} segundos ou ela será revertida",
+  "community_name_115a617": "Nome da comunidade",
   "config_4877f466": "Configuração",
+  "configure_your_network_7736471d": "Configure a sua rede",
+  "configure_your_new_community_network_a6daad12": "Configure sua nova rede mesh",
+  "confirm_6556b3a6": "Confirmar",
+  "confirm_location_2fe5ae11": "confirmar localização",
+  "congratulations_ffe43bf9": "Parabéns",
+  "connected_host_91e11459": "Host conectado",
+  "connection_fail_57f84354": "Não foi possível conectar a %{meta_ws}",
+  "count_days_de0c6a32": {
+      "one": "dia",
+      "other": "%{count} dias"
```

Comment:
%{count} will duplicate number rendering here.

It should be "dias" instead of "%{count} dias". The same for seconds, minutes and hours :)

Reply:
Following en.json or es.json format

Reply:
@luandro I've done this little fix!